Installing-Oppia-(Windows;-Python-3).md

```diff
@@ -1,5 +1,11 @@
 # Table of Contents
 
+> [!NOTE]
+> Most of scripts in Oppia are designed to support UNIX based systems, this includes Linix and MacOS but not Windows. So, for Windows we have 3 major methods to support this.
```

Comment:
Most of scripts --> Most scripts

UNIX based systems, this includes --> UNIX-based systems. These include

Linix --> Linux

Also, what does "this" at the end refer to? It's not quite clear, perhaps worth rephrasing. Maybe "to support running these scripts:"?